src/System.Data.Common/ref/System.Data.Common.Manual.cs

```diff
@@ -28,4 +28,52 @@ public abstract partial class DbParameterCollection : System.Collections.ICollec
         bool System.Collections.IList.IsReadOnly { get { return default(bool); } }
         bool System.Collections.ICollection.IsSynchronized { get { return default(bool); } }
     }
+
+    public abstract partial class DbColumn
+    {
+        public bool? AllowDBNull { get; protected set; }
+        public string BaseCatalogName { get; protected set; }
+        public string BaseColumnName { get; protected set; }
+        public string BaseSchemaName { get; protected set; }
+        public string BaseServerName { get; protected set; }
+        public string BaseTableName { get; protected set; }
+        public string ColumnName { get; protected set; }
+        public int? ColumnOrdinal { get; protected set; }
+        public int? ColumnSize { get; protected set; }
+        public bool? IsAliased { get; protected set; }
+        public bool? IsAutoIncrement { get; protected set; }
+        public bool? IsExpression { get; protected set; }
+        public bool? IsHidden { get; protected set; }
+        public bool? IsIdentity { get; protected set; }
+        public bool? IsKey { get; protected set; }
+        public bool? IsLong { get; protected set; }
+        public bool? IsReadOnly { get; protected set; }
+        public bool? IsUnique { get; protected set; }
+        public int? NumericPrecision { get; protected set; }
+        public int? NumericScale { get; protected set; }
+        public string UdtAssemblyQualifiedName { get; protected set; }
+        public Type DataType { get; protected set; }
+        public string DataTypeName { get; protected set; }
+        public virtual object this[string property]
+        {
+            get
+            {
+                return default(object);
+            }
+        }
+    }
+
+    public static class DbDataReaderExtension
```

Comment:
Should this be plural, e.g. `DbDataReaderExtensions`? As presumably additional extensions might be added in the future and the plural `Extensions` suffix is used elsewhere.